docker/docker-compose-aspnetcore/docker-compose.oats.yml

```diff
@@ -18,6 +18,11 @@ services:
     image: redis:7.2
     ports:
       - "6379:6379"
+    healthcheck:
```

Comment:
Please also add this to `docker-compose.self-contained.oats.yml`.

Reply:
Please also add it to `docker-compose.net8.oats.yml`.

Sorry forcing you to catch up with other stuff that was merged in the meantime ...

Reply:
Added these files to the csproj so they don't get missed again 👍